mne/preprocessing/tests/test_maxwell.py

```diff
@@ -25,11 +25,24 @@
                         'test_move_anon_raw_simp_stdOrigin_sss.fif')
 sss_nonstd_fname = op.join(data_path, 'SSS',
                            'test_move_anon_raw_simp_nonStdOrigin_sss.fif')
+sss_fine_cal_fname = op.join(data_path, 'SSS',
+                             'test_move_anon_raw_fineCal_sss.fif')
+
+fine_cal_fname = op.join(data_path, 'SSS', 'sss_cal_3053.dat')
+fine_cal_fname_3d = op.join(data_path, 'SSS', 'sss_cal_3053_3d.dat')
+
+int_order, ext_order = 8, 3
 
 
 @testing.requires_testing_data
 def test_maxwell_filter():
-    """Test multipolar moment and Maxwell filter"""
+    """Test multipolar moment and Maxwell filter
+
+    Notes
+    -----
+    Elekta MaxFilter uses single precision, so expect filtered results to
+    differ slightly.
```

Comment:
By convention we usually don't add notes to these functions, so just make this a `#` comment instead